i18n/en.json

```diff
@@ -184,7 +184,7 @@
   "report-fail-content": "Report for edit on $1 has failed. Looks like config is not define correctly.",
   "report-already-reported": "The user has been already reported.",
   "autoreport-title": "Auto report",
-  "autoreport-content": "$1 has been warned muliple times. Do you want to report them to Admin Noticeboard?",
+  "autoreport-content": "$1 has been warned multiple times. Do you want to report them to Admin Noticeboard?",
```

Comment:
I'm unsure if this should also be "to the Admin Noticeboard" instead. Someone with better English than mine should check that. Thanks.

Reply:
I hope 654392ccd8bdc081fa59b3977f6f48d9191df52e fixes those.